.travis.yml

```diff
@@ -25,7 +25,11 @@ install:
 before_script:
   - psql -c "create database datahub;" -U postgres
 
-script: 
+script:
+  # Migrations must be run before creating public users
+  - python manage.py makemigrations
```

Comment:
makemigrations should be unnecessary, since all necessary migrations should already be in source control.